examples/java/ml/yuuki/chris/rpc/examples/WindowTest.java

```diff
@@ -11,6 +11,10 @@
 	public static void main(String args[]) {
 		DiscordRPC lib = DiscordRPC.INSTANCE;
 		DiscordRichPresence presence = new DiscordRichPresence();
+		if (args.length == 0) {
```

Comment:
Why not move this to the first line? No point in loading the RPC library if its gonna fail right after.

Reply:
Ah that's very true. Nice catch 😄